### PR TITLE
Swift Package Manager

### PR DESCRIPTION
**Pull request by maxxfrazer**
added support for Swift Package Manager. Added import statements to any troublesome files in Sources/*

### Goals 🥅
- Add Swift Package Manager Support
This PR needs to be merged first:
https://github.com/huri000/QuickLayout/pull/32

### Implementation Details ✏️
- Added Package.swift
- Added import of relevant libraries in some source files
  - It seems Xcode 11 is more strict on these things

### Testing Details 🔍
- Used the Example project to see if anything had broken, haven't committed that yet as it'll make the PR messy (pod updates are included)

### Screenshot Links 📷
No visible changes

### CODE REVIEW

**Comment by maxxfrazer** [comment]
Thanks for merging the other PR!
Have changed the dependency to point to @huri000 
It currently points to the master branch on QuickLayout; once a release has been added it'll just need to be changed to something like:
`.upToNextMajor(from: "3.1.0")`

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+let package = Package(
+  name: "SwiftEntryKit",
+  platforms: [
+    .iOS(.v9)
+  ],
+  products: [
+    .library(name: "SwiftEntryKit", targets: ["SwiftEntryKit"])
+  ],
+  dependencies: [
+    .package(url: "https://github.com/huri000/QuickLayout", .branch("master"))
```
Comment (huri000) [inline Package.swift]: `.exact("3.0.2")` (or any other fitting expression) instead of `.branch("master")` and we're good to  🚀.

Comment (maxxfrazer) [reply Package.swift]: Looks like the tag hasn’t been pushed on the page I’ve linked below yet, but have made the changes 👍
https://github.com/huri000/QuickLayout/releases

Comment (maxxfrazer) [reply Package.swift]: I think it was meant to be 3.0.1, have made that change and pushed 👍

**Pull request merged**